cumulus_library_data_metrics/q_date_in_lifetime/q_date_in_lifetime.jinja

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE {{ study_prefix }}__q_date_in_lifetime_{{ src|lower }} AS
+{% for field in dates %}
+    SELECT
+        {{ src }}.{{ patient_field }}.reference as patient_ref,
+        P.patient_start AS date_of_birth,
+        P.patient_end AS date_of_death,
```

Comment:
nit: why have two different names for these concepts? I'd vote to either keep patient_start here or change the SQL to use date_of_birth, vs having two names.